src/features/gallery/Card.tsx

```diff
@@ -0,0 +1,225 @@
+// import useDnaInfo from '~/hooks/dna/useDnaInfo';
+import { useState } from 'react';
+import Image from 'next/image';
+import { css, type Theme } from '@emotion/react';
+
+import Softskill from '~/components/graphic/softskills/Softskill';
+import { type Softskills } from '~/components/graphic/softskills/type';
+import BookmarkIcon from '~/components/icons/BookmarkIcon';
+import Pill, { type Color } from '~/components/pill/Pill';
+import { CAREER_CARD_IMAGE_BY_GROUP } from '~/constants/dnaImage';
+import { type SurveyType, type TargetType } from '~/remotes/gallery';
+import { BODY_1, BODY_2_REGULAR, DETAIL, HEAD_1_BOLD, HEAD_3_SEMIBOLD } from '~/styles/typo';
+
+interface Props {
+  survey: SurveyType;
+  target: TargetType;
+  isMine?: boolean;
+}
+
+function Card(props: Props) {
+  // const { group = 'C' } = useDnaInfo(props.survey.survey_id);
+
+  const group = 'C';
+  const viewTendencies = props.survey.tendencies.slice(0, 3);
+
+  return (
+    <section css={sectionCss}>
+      <div css={topBoxCss}>
+        <div css={topInnerCss}>
+          <hgroup>
+            <h2>
+              {props.target.nickname}
+
+              {props.isMine && <span css={isMineCss}>ME</span>}
+            </h2>
+            <p>{props.target.job}</p>
+          </hgroup>
+          <div css={tagWrapperCss}>
+            {viewTendencies.map((tendency, idx) => (
+              <Pill color={COLOR_INDEX[idx]} key={tendency.name} css={tagItemCss}>
+                <Softskill name={tendency.name as Softskills} />
+                {tendency.name.replaceAll('_', ' ')}
+              </Pill>
+            ))}
+          </div>
+        </div>
+        <picture>
+          <source srcSet={CAREER_CARD_IMAGE_BY_GROUP[group].webp} type="image/webp" />
+          <Image
+            priority
+            unoptimized
+            src={CAREER_CARD_IMAGE_BY_GROUP[group].png}
+            alt="DNA 이미지"
+            width={334}
+            height={228}
+            css={imageCss}
+          />
+        </picture>
+      </div>
+      <div css={feedbackWrapperCss}>
+        <div>
+          <h3>
+            받은 피드백 <span>{props.survey.feedback_count}</span>
```

Comment:
```suggestion
{ survey }: Props

            받은 피드백 <span>{survey.feedback_count}</span>
```

분해할당을 해서 하지 않은 이유가 궁금해요!!

Reply:
부모에서 오는 props와 컴포넌트 내 state의 구분 외에는 별 다른 이유는 없었습니다!
-> 분해할당으로 변경했어요!